action.yml

```
@@ -38,11 +39,14 @@ runs:
         major_type=$(echo $TYPE | cut -d "," -f 1)
         if [[ $RUN_BRANCHES_ON_PREVIOUS_MAJOR == "false" ]] && [[ $major_type == "previous" ]]; then
           echo "Workflow is configured to not dispatch workflow for branches on previous major. Exiting."
+          echo "doDispatch=false" >> $GITHUB_OUTPUT
```

```suggestion
          echo "do_dispatch=false" >> $GITHUB_OUTPUT
```
And update for other references in this PR

snake_case for output variables looks like it's the current convention e.g.
- https://github.com/silverstripe/gha-ci/blob/1/.github/workflows/ci.yml#L860
- https://github.com/silverstripe/gha-pull-request/blob/1/action.yml#L47

---

Done